controllers/packagebundlecontroller_controller.go

```diff
@@ -147,12 +146,22 @@ func (r *PackageBundleControllerReconciler) Reconcile(ctx context.Context, req c
 	}
 
 	if !r.bundleManager.IsBundleKnown(ctx, bundles.Items, latestBundle) {
+		r.Log.V(6).Info("New latest bundle", "bundle", latestBundle.Name)
 		err = r.Client.Create(ctx, latestBundle)
 		if err != nil {
 			return result, fmt.Errorf("creating new package bundle: %s", err)
 		}
 	}
 
+	if pbc.Spec.ActiveBundle == "" {
+		r.Log.V(6).Info("Activating new bundle", "bundle", latestBundle.Name)
+		pbc.Spec.ActiveBundle = latestBundle.Name
+		err = r.Client.Update(ctx, pbc)
+		if err != nil {
+			return result, fmt.Errorf("updating active bundle: %s", err)
+		}
+		return result, fmt.Errorf("updating active bundle: %s", err)
```

Comment:
is this a typo? Don't we want to return nil in case the update is successful?

Reply:
Yeh, fixed thanks